validatorPassword.js

```diff
@@ -0,0 +1,68 @@
+// Must be a minimum of 8 characters
+function validatePassword (password) {
+    const lowerCase = 'abcdefghijklmnopqrstuvwxyz'
+    const upperLetter = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+    const numeric = '123456789'
+    const specialCharacter = '!@#$%^&*=+'
+    let isLongEnough = false
+    let hasLowerCase = false
```

Comment:
You did not use your ESLint. there are no spaces, indentations, blank lines where necessary.